### PR TITLE
Add table-view support for CLI list commands

### PR DESCRIPTION
**Pull request by feedmeapples**
**What changed?**
Added Table view to CLI list util in addition to JSON view.  Use --print_json to switch between them. Default is Table view.

**Why?**
Allows to switch between table and json view for CLI list commands.

**How did you test it?**
Ran tests and a list command that utilizes the new util. Example:
`./tctl admin shard list_tasks --shard_id 3 --task_type 3 --pagesize 4 --more --print_json --username <db_flags>`

**Potential risks**
Smaller risk of not properly handling a field of a row since the table view constructs the table dynamically using reflection

### CODE REVIEW

File: tools/cli/util.go [hunk 1]
```diff
@@ -904,25 +905,67 @@ func showNextPage() bool {
...
+		fields = append(fields, t.Field(i).Name)
+	}
+
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetBorder(false)
+	table.SetColumnSeparator("|")
+	table.SetHeader(fields)
+	table.SetHeaderLine(false)
+	for i := 0; i < len(items); i++ {
+		item := reflect.ValueOf(items[i]).Elem()
```
Comment (alexshtin) [inline tools/cli/util.go:959]: This will panic if `items[i]` is not a pointer but `string` for example.

File: tools/cli/util.go [hunk 2]
```diff
@@ -904,25 +905,67 @@ func showNextPage() bool {
...
+			pageItems = pageItems[:0]
+		}
+	}
+
+	return nil
+}
+
+func printTable(items []interface{}) error {
+	if len(items) == 0 {
+		fmt.Println("no items to print")
```
Comment (alexshtin) [inline tools/cli/util.go:942]: I think you should remove this message or rephrase it. User invokes CLI command, he doesn't ask to "print". Something like `<empty list>` would be better here. But I believe that empty output is the best UX.

File: tools/cli/util.go (hunk 1 above)
Comment (alexshtin) [reply tools/cli/util.go:959]: Another case is `***string`. Do you want to support it?

Comment (feedmeapples) [reply tools/cli/util.go]: addressed both

**Pull request merged**